keyboards/doio/kb04/info.json

```diff
@@ -0,0 +1,111 @@
+{
+    "manufacturer": "DOIO",
+    "keyboard_name": "KB04-01",
+    "maintainer": "filmstarr",
+    "bootloader": "stm32duino",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B5", "pin_b": "B6"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "cols": ["B14", "B13", "B12", "B0", "A7"],
+        "rows": ["B3"]
+    },
+    "processor": "STM32F103",
+    "rgb_matrix": {
+        "animations": {
+            "solid_color": true,
+            "alphas_mods": true,
+            "gradient_up_down": true,
+            "gradient_left_right": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "rainbow_moving_chevron": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_fractal": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "typing_heatmap": true,
+            "digital_rain": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [0, 0], "x": 0, "y": 0, "flags": 1},
+            {"matrix": [0, 1], "x": 75, "y": 0, "flags": 1},
+            {"matrix": [0, 2], "x": 149, "y": 0, "flags": 1},
+            {"matrix": [0, 3], "x": 224, "y": 0, "flags": 1},
+            {"matrix": [0, 4], "x": 112, "y": 64, "flags": 1},
+            {"matrix": [0, 5], "x": 112, "y": 64, "flags": 1},
+            {"matrix": [0, 6], "x": 112, "y": 64, "flags": 1},
+            {"matrix": [0, 7], "x": 112, "y": 64, "flags": 1}
```

Comment:
LED config is incorrect.

Reply:
If this was referring to
```json
{"matrix": [0, 4], "x": 112, "y": 64, "flags": 1},
{"matrix": [0, 5], "x": 112, "y": 64, "flags": 1},
{"matrix": [0, 6], "x": 112, "y": 64, "flags": 1},
{"matrix": [0, 7], "x": 112, "y": 64, "flags": 1}
```
still being there from when the imaginary pins were removed, that was resolved in https://github.com/qmk/qmk_firmware/pull/22754/commits/a03e4a27cc3b9b332011f2cbaaf1b0e654b40a32 and https://github.com/qmk/qmk_firmware/pull/22754/commits/b0f79e2255f682f257718338665443e9243a00f9, unless of course, you meant something else.

Reply:
![image](https://github.com/qmk/qmk_firmware/assets/4781841/d584d83e-08c5-4743-95e0-ec6769180b21)
🤷

Reply:
Just compiled both `default` and `via`  =just now and they were successful. QMK CI must be playing catch up now.